src/core.js

```diff
@@ -917,6 +917,21 @@ $.extend( $.validator, {
 				valid: true,
 				message: this.defaultMessage( element, "remote" )
 			});
+		},
+
+		// cleans up all forms and elements, removes validator-specific events
+		destroy: function() {
+			this.resetForm();
+
+			$( this.currentForm )
+				.off( "submit.validate" )
+				.off( "focusin.validate focusout.validate keyup.validate",
```

Comment:
This can be simplified to `".validate"` as the first argument, should do the same thing. I guess the second argument is still required, but would be good verify that.